app/apis/api_kubernetes.py

```diff
@@ -814,6 +814,7 @@ def switch_drucker_service_model_assignment(
             name="{0}-deployment".format(sobj.service_name),
             namespace=sobj.service_level
         )
+    response_body["status"] = True
```

Comment:
It might fix the problem, but meaning of `status` as `True` seems confusing.
`status` is often like `OK`, `NG` etc.

Did `response_body["success"] = True` not work?

Reply:
All backend responses return `response_body["status"] = True` now. If we change it to `response_body["success"] = True`, we need to replace all fields in backend and frontend. 

We may use `status` as integer, but may change in the next PR.

Reply:
Yes, a separate PR is better.

Reply:
Thank you :)